packages/nimble-components/src/wafer-map/types.ts

```diff
@@ -36,3 +36,15 @@ export interface WaferMapColorsScale {
     colors: string[];
     values: number[];
 }
+
+export interface Dimensions {
+    width: number;
+    height: number;
+}
+
+export interface Margin {
+    top: number;
+    right: number;
+    bottom: number;
+    left: number;
+}
```

Comment:
Looking at how these types are used (just calculated in the constructor of computations), I think we can mark all the members readonly to prevent accidental changes:
```suggestion
export interface Dimensions {
    readonly width: number;
    readonly height: number;
}

export interface Margin {
    readonly top: number;
    readonly right: number;
    readonly bottom: number;
    readonly left: number;
}
```